torchaudio/functional.py

```diff
@@ -376,7 +376,7 @@ def create_fb_matrix(
     up_slopes = slopes[:, 2:] / f_diff[1:]  # (n_freqs, n_mels)
     fb = torch.max(zero, torch.min(down_slopes, up_slopes))
 
-    if norm == "slaney":
+    if norm is not None and norm == "slaney":
```

Comment:
I think we should not allow values other than `"slaney"` or `None` to prevent the case where unintended value is passed, like typo.